auth-server/oauth-scim-service/src/main/resources/application.properties

```diff
@@ -120,17 +120,15 @@ cors.allowed.origins=${CORS_ALLOWED_ORIGINS}
-mail.account.locked.content=<html><body><div style='margin:20px;padding:10px;font-family: sans-serif;font-size: 14px;'><span>Hi</span><br/><br/><span>This is to inform you that, as a security measure, your user account has been temporarily locked<br/> for a period of ${oauth.scim.service.account.lockout.period.minutes} minutes, due to multiple consecutive failed sign-in attempts with incorrect password.<br/><br/><span>Please try signing in again after this period.</span><br/><br/><span>Alternatively, you could use the temporary password given below to sign in and change your password.</span><br/><br/><span><strong>Temporary Password: </strong>{{tempPassword}}</span><br/><br/><span>Please note that this temporary password can be used only once and is valid for a period of<br/>${oauth.scim.service.account.lockout.period.minutes} minutes only.</span><br/><br/><span style='font-size:15px;'>Thanks,</span><br/><span>The ${org.name} MyStudies support team<span><br/><span>-------------------------------------------------------------------------------------------------</span><br/><span style='font-size:10px;'>Note: This is an auto-generated email. Please do not reply.</span></div></body></html>
+mail.account.locked.subject=Your account has been locked
+mail.account.locked.content=<html><body><div style='margin:20px;padding:10px;font-family: sans-serif;font-size: 14px;'><span>Hi</span><br/><br/><span>This is to inform you that, as a security measure, your admin user account for the ${org.name} Participant Manager portal has been temporarily locked for a period of ${oauth.scim.service.account.lockout.period.minutes} minutes, due to multiple consecutive failed sign-in attempts with incorrect password.<br/><br/><span>Please try signing in again after this period.</span><br/><br/><span>Alternatively, use the temporary password given below to sign in to the Participant Managerand then set up a new password.</span><br/><br/><span><strong>Temporary Password: </strong>{{tempPassword}}</span><br/><br/><span>Please note that this temporary password can be used only once and is valid for a period of<br/>${oauth.scim.service.account.lockout.period.minutes} minutes only.</span><br/><br/><span>For any questions or assistance, please write to <a>${mail.contact-email}</a></span><br/><br/><span style='font-size:15px;'>Thanks,</span><br/><span>The ${org.name} MyStudies support team<span><br/><span>-------------------------------------------------------------------------------------------------</span><br/><span style='font-size:10px;'>Note: This is an auto-generated email. Please do not reply.</span></div></body></html>
```

Comment:
There are typos in here (for example "Managerand")

Reply:
Fixed review comment

Reply:
Typo is still here

Reply:
I have updated for [application.properties](https://github.com/GoogleCloudPlatform/fda-mystudies/blob/2.0.2-release-candidate/auth-server/oauth-scim-service/src/main/resources/application.properties) in 2.0.2-release-candidate branch